libs/application/templates/general-fishing-license/src/lib/GeneralFishingLicenseTemplate.ts

```diff
@@ -122,7 +122,7 @@ const GeneralFishingLicenseTemplate: ApplicationTemplate<
             shouldBeListed: true,
             shouldBePruned: true,
             // Applications that stay in this state for 24 hours will be pruned automatically
-            whenToPrune: 24 * 3600 * 1000,
+            whenToPrune: 10, // FOR TESTING
```

Comment:
Just in case: Was this supposed to change back before hotfix or is it ok to test on prod?